ci/minikube/test-conbench-on-mk.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+set -o errexit
```

Comment:
wow, verbose names, fancy!

Reply:
My brain refuses to learn which of those one-letter variants really means what (I think I got -e by now, but ...). I like the long versions more :).